src/test/java/org/broadinstitute/hellbender/utils/read/ReadUtilsUnitTest.java

```diff
@@ -3,8 +3,19 @@
 import com.google.api.services.genomics.model.LinearAlignment;
 import com.google.api.services.genomics.model.Position;
 import com.google.api.services.genomics.model.Read;
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
 import htsjdk.samtools.*;
+import htsjdk.samtools.SamReader.PrimitiveSamReader;
+import htsjdk.samtools.cram.ref.CRAMReferenceSource;
+import htsjdk.samtools.cram.ref.ReferenceSource;
 import htsjdk.samtools.reference.IndexedFastaSequenceFile;
+import htsjdk.samtools.util.CloseableIterator;
+import java.io.IOException;
+import java.io.InputStream;
```

Comment:
Quite a few of these imports are unused.

Reply:
Removed all unused imports in this file.

Reply:
Since unused imports are an issue, I ran the IDE tool to remove all unused imports. The source should be easier on the eyes now.

Reply:
I've reverted all of the other files because this tool touched pretty much every single file in the project, that was crazy.